src/Nethermind/Nethermind.AuRa.Test/AuRaBlockProducerTests.cs

```diff
@@ -97,8 +98,9 @@ public void InitProducer(IAuraConfig auraConfig)
                             Timestamper,
                             LimboLogs.Instance,
                             AuRaStepCalculator,
+                            NullReportingValidator.Instance, 
                             auraConfig,
-                            NodeAddress);
+                            new Signer(0, TestItem.PrivateKeyA));
```

Comment:
would look nicer if you wrote ChainId.Olympic (and you probably wanted ChainId.Mainnet)
0 chain ID is Olympic which was a premainnet testnet and had some differences